content/architecture/index.textile

```diff
@@ -0,0 +1,141 @@
+---
+title: Architecture overview
+meta_description: "Learn more about Ably's platform architecture."
+meta_keywords: "architecture"
+---
+
+Ably's platform architecture is built to deliver dependable realtime experiences at global scale.
+
+As the definitive realtime experience platform, we serve billions of devices monthly and deliver over half a trillion messages monthly.
+
+h3. At a Glance
```

Comment:
I think most of the h3 should be h2

Reply:
Fixed in [59617e3](https://github.com/ably/docs/pull/2496/commits/59617e315db0bd03de9ff940fe6dcc45928d470d)